src/app/workspaces/workspace-details/workspace-details.controller.js

```diff
@@ -55,9 +55,10 @@ class WorkspaceDetailsCtrl {
       this.updateWorkspaceData();
     }
 
-
+    this.startUpdateWorkspaceStatus();
```

Comment:
This method only updates the status received with other details,
maybe, it would be better to begin updating status after workspace details will be loaded?